test/e2e/network/pod_lifecycle.go

```diff
@@ -68,7 +70,7 @@ var _ = common.SIGDescribe("Connectivity Pod Lifecycle", func() {
 
 		ginkgo.By("Try to connect to the webserver")
 		// Wait until we are able to connect to the Pod
-		podIPAddress := net.JoinHostPort(webserverPod.Status.PodIP, strconv.Itoa(80))
+		podIPAddress := net.JoinHostPort(webserverPod.Status.PodIP, strconv.FormatInt(longNetworkGracePeriod, 10))
```

Comment:
you are replacing a port by a graceperiod, the only thing neede to change here is to put 80 in a variable and use that

Reply:
oops, :/ sorry, fixed!